mllib/src/test/scala/org/apache/spark/mllib/linalg/UDTSerializationBenchmark.scala

```diff
@@ -24,7 +24,9 @@ import org.apache.spark.sql.catalyst.encoders.ExpressionEncoder
  * Serialization benchmark for VectorUDT.
  * To run this benchmark:
  * {{{
- * 1. without sbt: bin/spark-submit --class <this class> <spark mllib test jar>
+ * 1. without sbt:
+ *    bin/spark-submit --class <this class>
+ *      --jars <spark core test jar> <spark mllib test jar>
```

Comment:
--jars need includes `spark core test jar` because BenchmarkBase defined in this module.